onnx/examples/check_model.ipynb

```diff
@@ -92,8 +92,10 @@
    ],
    "source": [
     "# Check the model\n",
-    "onnx.checker.check_model(onnx_model)\n",
-    "print('The model is checked!')"
+    "try:\n",
+    "    onnx.checker.check_model(onnx_model)\n",
+    "else:\n",
+    "    print('The model is valid!')"
```

Comment:
Missing the following code?
```
except onnx.checker.ValidationError as e:
    print('The model is invalid: %s' % e)
```

Reply:
Previous to this change, and now, the behavior is the same: if the model is invalid, an exception will be raised that makes that obvious.
So I didn't see a need to catch the exception.
If you prefer I can add the catch, but could you please explain why?

Reply:
Thank you for the explanation. Actually the CIs will validate .ipynb and so a try needs a except.